website/public/locales/en/language.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Let's delete the last sentence in this message and then i'll merge.

Reply:
Done